src/auth/auth.service.ts

```diff
@@ -33,4 +38,28 @@ export class AuthService {
 
     return this.userService.create(userDto);
   }
+
+  async login(user: JwtUser) {
+    const token = this.jwtService.sign(user);
```

Comment:
should only sign minimal information to validate user, and it must not contains confidential personal informations. The JWT token is not encrypted, contains any confidential information in payload would be a serious security issue.

Reply:
I get that. I deleted the email field.
[fix: delete email field in jwt payload](https://github.com/Cophr/backend/pull/10/commits/42a70d180967359650700fc909aa6f2cf06dfdb9)